app/javascript/components/patient/PatientPage.js

```diff
@@ -16,10 +16,15 @@ class PatientPage extends React.Component {
           </Card.Header>
           <Card.Body>
             <Patient
+              current_user={this.props.current_user}
               jurisdiction_path={this.props.jurisdiction_path}
+              jurisdiction_paths={this.props.jurisdiction_paths}
               details={{ ...this.props.patient, blocked_sms: this.props.blocked_sms }}
               collapse={this.props.can_modify_subject_status}
               edit_mode={false}
+              follow_up_reasons={this.props.follow_up_reasons}
+              other_household_members={this.props.other_household_members}
+              display_follow_up_flag={this.props.current_user.role != 'enroller'}
```

Comment:
instead of this prop you can probably just use `can_modify_subject_status` and pass that in